### DISCUSSION TITLE
Sorting My Profile components

### DISCUSSION THREAD

**Discussion by clickersmudge** [Jacobtims/filament-breezy#347] [Q&A]
Hi,
Can someone help me sort the components?
Custom ProfilePrivacy component always loads right after ProfileEdit.

**app/Providers/Filament/AppPanelProvider.php**
```
->plugins([
                \BezhanSalleh\FilamentShield\FilamentShieldPlugin::make(),
                BreezyCore::make()
                    ->myProfile()
                    ->myProfileComponents([ProfileEdit::class, TwoFactorAuthentication::class, ProfilePrivacy::class])
                    ->withoutMyProfileComponents([
                        'personal_info',
                    ]),
            ])
```

I don't know how to set the sorting at the end.
https://github.com/jeffgreco13/filament-breezy?tab=readme-ov-file#sorting-my-profile-components

**Comment by njugunamwangi** [comment]
You can set it in the appserviceprovider like so

TwoFactorAuthentication::setSort(1)
